### DISCUSSION TITLE
Some completed Katas are still listed as "Unfinished".

### DISCUSSION THREAD

**Discussion by rtyping** [codewars/codewars.com#3219] [General]
I took six Katas, three Kata are in the "Completed" section, the remaining three are still in the "Unfinished" section. How to fix it?

**Comment by rtyping** [comment]
I clicks Test, all testcases are green, but the kata is left in the Unfinished section. Is it a codewars bug or I do something wrong?

**Reply by hobovsky** [reply]
What is your Codewars user name? Can you give a link to your CW profile?

Do you use multiple browsers or browser tabs to train on a single kata in a given moment?

**Reply by rtyping** [reply]
Thanks for the answer. I have already found the reason. That was my fault. I used the "Attempt" button to check the result and the "Test" button for final testing. If you use the "Attempt" button then the "Test" button, a "Submit" button will appear. I must have missed this feature in the manual.
